docs/synthetics/ci-cd/ci-cd-installation.md

```diff
@@ -0,0 +1,283 @@
+    - If you can't see the **Settings** tab for your repository, then you might not have permissions to edit them - contact your repository's administrator to take care of this step.
+
+![CI/CD GitHub Secret](/docs/synthetics/ci-cd/images/ci-cd-github-secret.png)
+
+
+### Usage
+
+Add the following to your GitHub workflow file:
+
+```yaml
+steps:
+  - name: Run Sematext Synthetics Check
+    uses: sematext/synthetics-cicd-action@v1
+    with:
+      MONITOR_GROUP_ID: 42
+      REGION: "US"
+      SEMATEXT_API_KEY: ${{ secrets.SEMATEXT_API_KEY }}
+      TARGET_URL: 'https://your-deployment-url.com'                 # Pass dynamically
+      GIT_COMMIT_HASH: '5a24a0f8cd48be7f315787dcc23ad418ecdb36f2'   # Pass dynamically
+      USE_HEAD_SHA: false
+```
+
+
+### Inputs
+
+| Input | Description | Required | Default |
+|-------|-------------|----------|---------|
+| `MONITOR_GROUP_ID` | The ID of the monitor group to run the checks against (e.g., `42`) | Yes | - |
+| `REGION` | The region where your Sematext Cloud account is located (`EU` or `US`) | Yes | - |
+| `SEMATEXT_API_KEY` | Your Sematext Cloud account's API key | Yes | - |
+| `TARGET_URL` | The URL to run the Synthetics check against | No | - |
+| `GIT_COMMIT_HASH` | The commit hash that the Synthetics check will be triggered for | No | - |
+| `USE_HEAD_SHA` | Whether to use the HEAD SHA for the Synthetics check | No | `false` |
+
+> **Note**: Either `GIT_COMMIT_HASH` must be provided or `USE_HEAD_SHA` must be set to `true`.
+
+
+### Outputs
+
+| Output | Description |
+|--------|-------------|
+| `result` | The full result output from the Synthetics check |
+| `status` | The status of the check (`passed` or `failed`) |
+| `error` | The error message if the check failed |
+| `group_run_url` | The URL of the Group Run in the Sematext Cloud UI |
+
+
+
+## Examples
+
+Here are some examples which illustrate how the action can be used depending on how complex your deployment setup is.
+
+
+### Workflow for a simple setup
+
+If your setup is such that you create deployments within GitHub (like through automatic deployment tools such as Vercel), you can make the workflow for this action trigger on the `deployment_status` event and simply configure the variables needed for the action:
+
+```yaml
+name: Trigger Sematext Synthetics Checks
+
+on:
+  deployment_status
+
+permissions:
+  contents: read   # Allows us to read the URL from the deployment event
+
+jobs:
+  sematext_synthetics_check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get Vercel Deployment URL
+        id: vercel_deployment_url
+        run: |
+          echo "DEPLOYMENT_URL=$(cat "$GITHUB_EVENT_PATH" | jq .deployment_status.target_url)" >> $GITHUB_ENV
+
+      - name: Run Sematext Synthetics CI/CD Integration
+        id: sematext_check
+        uses: hssalman/cicd-repo-action@v0.9.3
```

Comment:
@HSSalman is this the actual thing one will have to use or will this become sematext/.... ?

Reply:
It'll be an official `sematext` repo, yes. This is just a temporary one until we're satisfied with how things work and I left it in so that Fulya can easily try out these install guides.